From d0b696db78723362b0b963174989fe4b9c4f4ca6 Mon Sep 17 00:00:00 2001
From: zzc <zzc@rock-chips.com>
Date: Wed, 22 Mar 2017 16:08:41 +0800
Subject: [PATCH] arm64: rockchip_linux_defconfig: enable rtl8188eu

CONFIG_WIFI_LOAD_DRIVER_WHEN_KERNEL_BOOTUP=y
CONFIG_RTL8188EU=y

Change-Id: I5d19e92280f0fb91701a3f2547251069380593ef
Signed-off-by: zzc <zzc@rock-chips.com>
---
 arch/arm64/configs/rockchip_linux_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/configs/rockchip_linux_defconfig b/arch/arm64/configs/rockchip_linux_defconfig
index 9a5d8eece790..543d625c5877 100644
--- a/arch/arm64/configs/rockchip_linux_defconfig
+++ b/arch/arm64/configs/rockchip_linux_defconfig
@@ -201,7 +201,9 @@ CONFIG_HOSTAP=y
 CONFIG_HOSTAP_FIRMWARE=y
 CONFIG_HOSTAP_FIRMWARE_NVRAM=y
 CONFIG_WL_ROCKCHIP=y
+CONFIG_WIFI_LOAD_DRIVER_WHEN_KERNEL_BOOTUP=y
 CONFIG_AP6XXX=y
+CONFIG_RTL8188EU=y
 CONFIG_MWIFIEX=y
 CONFIG_MWIFIEX_SDIO=y
 # CONFIG_INPUT_MOUSEDEV is not set
-- 
2.35.3

